OnlineFlightSearchAPI/Services/FlightServices/ISearchFlightService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using OnlineFlightSearchAPI.Models;
+using System;
+using System.Collections.Generic;
+
+namespace OnlineFlightSearchAPI.FlightServices
+{
+    public interface ISearchFlightService
```

Comment:
Any reason for defining a service for a specific functionality? IFlightService seems more appropriate.

Reply:
Thats for ISP principle. In future if other functionalities comes like BookFlightService , CancelFlightService like that.

Reply:
Interface is fine, but why the functionality specific implementation?

Reply:
Yep renamed class as per suggestion